static/pages/partnerships/partnerships.html

```diff
@@ -235,7 +243,10 @@ <h3 class="md-display-1 partnerships-page-content-title">
         </md-card-content>
       </div>
     </md-card-content>
-    <md-card-content class="oppia-foundation-no-padding
+  </md-card>
+  <md-card layout-fill
```

Comment:
Just want to clarify your intention here.

Do you want the student image and the student quote to be its own card(independent section) or do you just want to highlight the image and the quote?

When I received Jordan and Rachel mock, I was under the impression that the image + the quote + the rest of the "We are growing..." is one coherent md-card.
Now it looks like the student and the image is a stand-alone md-card, I would have to tweak its padding (to have 80px top and bottom padding)

Reply:
Yep, I changed it to make the student+image more of a "call-out". I noticed on the homepage that the green background was used for short small things we wanted to highlight, so thought I would do that here, too.

So yep, intention is for the student image + quote to be its own card. In the older version it was a bit unclear how it was delineated from the section below.